### PR TITLE
Update to ROM 1.0.0

### PR DESCRIPTION
**Pull request by hmadison**
Clean up some tests and fix the setup for tests. Removed the `test/integration/relation_test.rb` test, since its testing behavior tested in rom core.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 gemspec
+
+gem 'pry'
```
Comment (franckverrot) [inline Gemfile]: Do you mind reverting those changes? I don't think (?) we need them.

Comment (hmadison) [inline Gemfile]: Cleaned that up.

**Comment by franckverrot** [comment]
Thanks for much for this contribution!

Is there a reason why you've changed double quotes to single quotes everywhere? Is it because of a suspected performance penalty related to double-quoted strings?

**Comment by hmadison** [comment]
@franckverrot Thats a rubocop autocorrect pass based off on the config from `rom-rb/rom`. Brings the style in line with the rest of the code on the org.

**Comment by franckverrot** [comment]
I won't argue (even if rubocop might be wrong on this one), and it's an org project now so... ¯_(ツ)_/¯

**Comment by franckverrot** [comment]
Thanks a lot @hmadison

**Pull request merged**